packages/vuetify/src/components/VCalendar/VCalendarCategory.ts

```diff
@@ -61,25 +72,69 @@ export default VCalendarDaily.extend({
           return this.getCategoryScope(this.getSlotScope(day), scope.category)
         }),
       }, [
-        getSlot(this, 'category', scope) || this.genDayHeaderCategoryTitle(scope.category),
+        getSlot(this, 'category', scope) || this.genDayHeaderCategoryTitle(scope.category && scope.category.categoryName),
         getSlot(this, 'day-header', scope),
       ])
     },
-    genDayHeaderCategoryTitle (category: string) {
+    genDayHeaderCategoryTitle (categoryName: any) {
```

Comment:
```suggestion
    genDayHeaderCategoryTitle (categoryName: string | null) {
```
(or whatever type it takes)